zed_ws/README.md

```diff
@@ -27,15 +28,39 @@ colcon build --symlink-install
 ros2 launch zed_wrapper zed_camera.launch.py camera_model:=zed2i
 ```
 
-> For simulation mode, append the flag `sim_mode:=true` at the end.
-
 ## Rviz visualization
 
 ```bash=
 ros2 launch zed_display_rviz2 display_zed_cam.launch.py camera_model:=zed2i
 ```
 
-> For simulation mode, append the flag `sim_mode:=true` at the end.
+## Simulation
+
+Append the flag `sim_mode:=true` at the end to enable the simulation mode. Use the flag `sim_platform` to specify the platform you want to simulate. The choices are `isaac_sim` and `gazebo`.
+
+```bash=
+# Simulation with Gazebo
+ros2 launch zed_display_rviz2 display_zed_cam.launch.py camera_model:=zed2i sim_mode:=true use_sim_time:=true sim_platform:=gazebo
+
+# Simulation with Isaac Sim
+ros2 launch zed_display_rviz2 display_zed_cam.launch.py camera_model:=zedx sim_mode:=true use_sim_time:=true sim_platform:=isaac_sim
+```
+
+> If you opt to simulate with Isaac Sim, you can utilize the provided Docker container. To launch Isaac Sim within the container, please follow the commands below:
+> ```bash=
+> # Outside the container
+> cd ~/ros2-agv-essentials/zed_ws/docker
+> docker compose build zed-isaac-sim
```

Comment:
I believe setting up a free NGC API key is required before this step

Ref: https://docs.omniverse.nvidia.com/isaacsim/latest/installation/install_container.html#container-deployment

Consider adding this link in the README.